codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -225,11 +232,13 @@ thickRectangle lw w h = ThickRectangle callStack lw w h
 
 -- | A thin circle, with this radius
 circle :: HasCallStack => Double -> Picture
-circle = arc 0 (2 * pi)
+circle r = Circle callStack r 
```

Comment:
You can simplify this to:

```haskell
circle = Circle callstack
```

Reply:
is this the eta reduction? :D

Reply:
Yes :)

The general trick is that if you define some function like:

```haskell
f x = g x
```

... you can simplify it to:

```haskell
f = g
```

For example, if you define:

```haskell
incrementEveryNumber numbers = map (+ 1) numbers
```

... you can simplify that to:

```haskell
incrementEveryNumber = map (+ 1)
```